Documents/AnalyticsToolDesignDocument.md

```diff
@@ -0,0 +1,207 @@
+# VFS For Git: Analytics Tooling Design Document
+
+This document details the design of and plan for the **VFS4G analytics tool**. The tool aims to help users identify reasons why their enlistment may be slowing down and the `git` commands running slowly on a VFS4G mounted repository. By collecting and reporting statistics, the power to uncover and hopefully fix problems as they arrive moves to the user, empowering them to use the software to its fullest potential.

```

Comment:
High level question: 
Do we use the word 'enlistment' to describe a `gvfs clone`? In my mind, enlistment evokes the world/tooling that our internal customers used before VFS4G. 

I have been trying to refer to them as 'clones', but I've been sloppy about it for sure :)

Reply:
If I'm going to be honest its definitely just something that I totally picked up from listening to other people. I think Jameson has been using it to refer to an instance of a repository and as I've been most closely working with him that's the meaning I've taken to it as well. Is there a concrete meaning for the word that has been decided? I've already considered and I think Jameson agrees with potentially adding a glossary to define a few things. Perhaps "hydration" may also belong there, though I did define it as best I could in the document. I'd love to hear your feedback about that!